bootloaders/riotboot/Makefile

```diff
@@ -0,0 +1,25 @@
+# Default RIOT bootloader
+APPLICATION = riotboot
+
+# Default testing board
+BOARD ?= samr21-xpro
+
+# Select the boards with riotboot feature
+FEATURES_REQUIRED += riotboot
+
+# Disable unused modules
+CFLAGS += -DNDEBUG -DLOG_LEVEL=LOG_NONE
+DISABLE_MODULE += auto_init
+
+# Include riotboot flash partition functionality
+USEMODULE += riotboot_slot
+
+# RIOT codebase
+RIOTBASE ?= $(CURDIR)/../../
+
+include $(RIOTBASE)/Makefile.include
+
+# limit riotboot bootloader size
+# TODO: Manage to set this variable for boards which already embed a
+# bootloader, currently it will be overwritten
+ROM_LEN := $(RIOTBOOT_LEN)
```

Comment:
Why the `:=`?

Reply:
This variable needs to be evaluated at the moment of its parsing, since the value is taken as a reference for further calculations. Otherwise the value changes and the results are non-consistent.